### PR TITLE
Feature/107/feedback

### PR DESCRIPTION
**Pull request by mhasilau**

### CODE REVIEW

File: src/app/components/appoint-interview-popup/appoint-interview-popup.component.ts
Comment (ozhurova) [inline src/app/components/appoint-interview-popup/appoint-interview-popup.component.ts:47]: delete console.log

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (ozhurova) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:16]: import this URL from environment.ts

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (ozhurova) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:18]: Are maybe make an interface and specify here { } better?

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (ozhurova) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:159]: delete console.log

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (ozhurova) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:214]: better done without setTimeout

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (ozhurova) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:137]: delete all console.log in the function

File: src/app/models/user.model.ts
Comment (ozhurova) [inline src/app/models/user.model.ts:9]: pls, chage roles: string[]

File: src/app/models/user.model.ts
Comment (ozhurova) [inline src/app/models/user.model.ts:30]: please change as it was before

**Review by ozhurova — CHANGES_REQUESTED** [review]

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (Nicolai-boiko) [reply src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:214]: Mb will better to use our auth service or angular httpclient.

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (Nicolai-boiko) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:3]: Mb it might be better to use our api service or angular httpclient.

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (mhasilau) [reply src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:214]: we need async for hand over to other component

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (mhasilau) [reply src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:3]: may be, but axios better for me

File: src/app/components/appoint-interview-popup/appoint-interview-popup.component.ts
Comment (vstrabko) [inline src/app/components/appoint-interview-popup/appoint-interview-popup.component.ts:55]: Is it really hard to add info func here instead of console log?

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.html
Comment (vstrabko) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.html:24]: Empty lines here looks a bit strange here.
Don't highlight div's content with empty lines.

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.html
Comment (vstrabko) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.html:65]: `>` should either be at the end of the previous line (as it is everywhere in the project) or aligned with opening `<`

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.html
Comment (vstrabko) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.html:89]: Make it one empty line.

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.scss
Comment (vstrabko) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.scss:28]: Make it one empty line.

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (vstrabko) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:217]: Do we really need to console log here?
May be info pop up?

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.html
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.html:5]: Why do you have user.name here?

File: src/app/components/login-form/login-form.component.ts
Comment (vstrabko) [inline src/app/components/login-form/login-form.component.ts:29]: Delete this console.log

**Review by vstrabko — CHANGES_REQUESTED** [review]
Pay more attention to html/css format and style.
Lot's of `console.log`s left in place in the code.

File: src/assets/locales/ru.json
Comment (vstrabko) [inline src/assets/locales/ru.json:156]: Bracket here not aligned.

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:244]: remove todo

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:255]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:218]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:217]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:214]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:211]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:206]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:203]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:200]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:199]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:196]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:193]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:192]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:189]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:185]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:164]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:163]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:158]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:82]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:80]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:65]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:61]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:152]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:167]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:221]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:224]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:248]: Replace any with something meaningful

File: src/app/components/login-form/login-form.component.ts
Comment (rastorc3v) [inline src/app/components/login-form/login-form.component.ts:23]: Replace any with something meaningful

File: src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts
Comment (rastorc3v) [inline src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts:72]: Add access modifier

**Review by rastorc3v — CHANGES_REQUESTED** [review]

File: src/assets/locales/ru.json
Comment (mhasilau) [reply src/assets/locales/ru.json:160]: change is gives opportunity to change
save is PUT into backend

**Review by Nicolai-boiko — APPROVED** [review]

**Review by ozhurova — APPROVED** [review]

**Review by vstrabko — APPROVED** [review]
Some things are fixed.
**Please wait for Vlad's approval before merging.**

**Review by rastorc3v — APPROVED** [review]

**Pull request merged**
